### DISCUSSION TITLE
Automatically add "from", "to" and "intervals" to any request

### DISCUSSION THREAD

**Discussion by dashie** [grafana/grafana-infinity-datasource#1061] [Ideas]
I would like to be able to define URL query param for all the requests and pass "from", "to" and "intervals" without to specify them on any query

**Comment by yosiasz** [comment]
I don't think you can. I believe you need to explicitly add them as query parameters

**Comment by yesoreyeram** [comment]
Most of the APIs will fail if they receive unexpected paramaters / unexpected format. Also not the api require paramenters in query string, they need to be in body and in other places.

So this is no unless there is a strong justification.
